compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -797,6 +806,18 @@ prettyNodeNode nn = do
         ppChildren <- prettyChildren node_RollbackChildren
         pure $ keyword_ "rollback" $$ ppChildren
 
+-- | Take a list of parties and the singular and multiple present tense verbs
+-- Depending on the count of parties, returns a prettified list of these elements, using @,@ and @and@, as well as the correct verb
+-- e.g. @[a, b, c] -> a, b and c@
+partiesAction :: V.Vector Party -> String -> String -> (Doc SyntaxClass, Doc SyntaxClass)
+partiesAction pv singular multiple = go $ mapV prettyParty pv
```

Comment:
more a style thing than anything else; `go` makes me think there's going to be recursion, I think here a simple `case` on `mapV prettyParty pv` would be easier to follow.

Reply:
yeah thats fair, i was considering calling it aux, but i realise it doesn't really need to be its own thing